src/components/Details.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { useLocation } from 'react-router-dom';
+
+const Details = () => {
+  const location = useLocation();
+  const { state } = location;
+  return (
+    <section className="container details-card">
+      <h2 className="title">
+        {state.symbol}
+      </h2>
+      <div>
+        <span>
+          Volume:
+          {state.volume}
+        </span>
+        <span>
+          Ask price:
+          {state.askPrice}
+        </span>
+        <span>
+          Price change:
+          {state.priceChange}
+        </span>
+        <span>
+          Last price:
+          {state.lastPrice}
+        </span>
+        <span>
+          Count:
+          {state.count}
+        </span>
+        <span>
+          Bid price:
+          {state.bidPrice}
+        </span>
+        <span>
+          Open time:
+          {state.openTime}
+        </span>
+        <span>
+          Close time:
+          {state.closeTime}
+        </span>
+      </div>
+    </section>
+  );
+};
+
```

Comment:
- [ ] As mentioned by the previous reviewer, kindly make sure your design stick as much as possible to the one from the given template. The **Homepage** and the **Detail** pages should have a **banner section** similar to the design below:
![Screenshot from 2022-09-09 20-22-26](https://user-images.githubusercontent.com/83042742/189418723-00def865-3294-48c9-b2b0-9d70d64da2b7.png)